castle/cms/caching/__init__.py

```diff
@@ -247,11 +265,25 @@ def purge(self):
                 for url in getURLsToPurge(path, settings.cachingProxies):
                     purger.purgeAsync(url)
 
-        success = True
+        success = []
         if cf.enabled:
             self.cf_enabled = True
             resp = CastlePurger.purgeSync(urls, cf)
-            success = resp.json()['success']
+            success.append({'name': 'Cloudflare', 'value': resp.json()['success']})
```

Comment:
Maybe want a try/except here in case resp is a 4xx or 5xx and has no json() or no success key